### ISSUE TITLE
Coverage reports are not generated

### ISSUE THREAD

**Issue by Blackbaud-BobbyEarl** [blackbaud/skyux-sdk-builder#193]
Relevant: 
- https://github.com/gotwarlost/istanbul/issues/925
- https://github.com/wycats/handlebars.js/issues/1636

```
Handlebars: Access has been denied to resolve the property "statements" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "branches" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "functions" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "lines" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
```

**Comment by springfred** [comment]
I have the same issue.

**Comment by Blackbaud-BobbyEarl** [comment]
I'm not sure if your problem is specific to our project @springfred; however, you'll need to update `istanbul-reports` to version `2.2.7` or the `3.x` major version.

If you are using our project (`@skyux-sdk/builder`), I'm releasing a version shortly that will contain https://github.com/blackbaud/skyux-sdk-builder/pull/194/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R86

**Comment by Blackbaud-BobbyEarl** [comment]
Addressed in [`3.13.0`](https://github.com/blackbaud/skyux-sdk-builder/blob/master/CHANGELOG.md#3130-2020-01-15)
